activesupport/lib/active_support/duration.rb

```diff
@@ -422,6 +422,11 @@ def eql?(other)
       Duration === other && other.value.eql?(value)
     end
 
+    # Returns +true+ if the duration is less than 0 seconds.
+    def negative?
+      @value < 0
+    end
```

Comment:
Can you also benchmark using `delegate :negative?, to: :@value`?

This will be slower than your solution, but I have an idea how to optimize it that would allow to more easily eagerly define much more than just `negative?` without lots of extra code.